src/rand.rs

```diff
@@ -66,11 +66,20 @@ fn xor_psuedo_rng_u8_alphanumerics(rand_fn: &dyn Fn() -> u32) -> Vec<u8> {
     r
 }
 
+fn xor_pseudo_rng_u8_bytes(rand_fn: &dyn Fn() -> u32) -> Vec<u8> {
+    let mut r = vec![];
+
+    for n in &rand_fn().to_ne_bytes() {
+        r.push(*n);
+    }
+    r
```

Comment:
This can also be written as an 1-liner: `rand_fn().to_ne_bytes().to_vec()`

Reply:
Aah yes!